gadget/validate_test.go

```diff
@@ -305,49 +470,10 @@ volumes:
 	err = os.Mkdir(fooPath, 0755)
 	c.Assert(err, IsNil)
 	// validate should no longer complain
-	err = gadget.Validate(s.dir, nil, nil)
+	err = gadget.ValidateContent(ginfo, s.dir)
 	c.Assert(err, IsNil)
 }
 
-func (s *validateGadgetTestSuite) TestValidateClassic(c *C) {
```

Comment:
this test too seems to be MIA

Reply:
I see equivalent tests are already in gadget_test.go so this is fine that this test goes away